hub-config/tasks.json

```diff
@@ -1,25 +1,224 @@
 {
-    "schema_version": "https://raw.githubusercontent.com/Infectious-Disease-Modeling-Hubs/schemas/main/v2.0.0/tasks-schema.json",
-    "rounds": [
+  "schema_version": "https://raw.githubusercontent.com/hubverse-org/schemas/main/v3.0.1/tasks-schema.json",
+  "rounds": [
+    {
+      "round_id_from_variable": true,
+      "round_id": "nowcast_date",
+      "model_tasks": [
         {
-            "round_id_from_variable": true,
-            "round_id": "origin_date",
-            "model_tasks": [
-                {
-                    "task_ids": {
-                        "origin_date": {
-                            "required": [],
-                            "optional": []
-                        }
-                    },
-                    "output_type": {},
-                    "target_metadata":[]
-                }
-            ],
-            "submissions_due": {
-                "start": "1900-01-01",
-                "end": "2900-12-31"
+          "task_ids": {
+            "nowcast_date": {
+              "required": [
+                "2024-09-11"
+              ],
+              "optional": null
+            },
+            "target_date": {
+              "required": null,
+              "optional": ["2024-09-11", "2024-09-04", "2024-08-28", "2024-08-21"]
+            },
+            "location": {
+              "required": null,
+              "optional": ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE", "DC", "FL", "GA", "HI", "ID", "IL", "IN", "IA", "KS", "KY", "LA", "ME", "MD", "MA", "MI", "MN", "MS", "MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY", "NC", "ND", "OH", "OK", "OR", "PA", "RI", "SC", "SD", "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI", "WY", "PR"]
+            },
+            "variant": {
+              "required": ["24A", "24B", "24C", "other", "recombinant"],
+              "optional": null
             }
+          },
+          "output_type": {
+            "mean": {
+              "output_type_id": {
+                "required": null,
```

Comment:
I always have to read and re-read the rules about which property to set to `null` and which to `"NA"`. But I think this is the right set-up here to have `"mean"` output_type be optional. 
<img width="513" alt="image" src="https://github.com/user-attachments/assets/5254f5bc-454e-4870-bdc3-56f4efc7352b">